os/board/esp32_DevKitC/src/esp32_bringup.c

```diff
@@ -0,0 +1,107 @@
+/******************************************************************
+ *
+ * Copyright 2018 Samsung Electronics All Rights Reserved.
```

Comment:
Please set a year 2019, not 2018.

Reply:
For all of files

Reply:
OK, all the new files in this commit are fixed, thanks.

Reply:
Not fixed yet. Please check it again.